packages/api-v2/src/APIProvider.tsx

```diff
@@ -25,6 +25,7 @@ type TSubscribeFunction = <T extends TSocketSubscribableEndpointNames>(
 type TUnsubscribeFunction = (id: string) => void;
 
 type APIContextData = {
+    customLoginIDKey?: string;
```

Comment:
API providere shoulnd't be aware login id (its just providing api). LoginId is within AuthProvider responsibility. 
Can we parametrize only AuthProvider and just leave APIprovider without changes? (hopefully that should be small change to this pr - just move parameters to AuthProvider)

Reply:
If `APIProvider` was to be made `loginid`-agnostic, `getEnvironment ` and `switchEnvironment ` would have to be moved to `AuthProvider` too

Reply:
yes, they should, 
to be precise: they should be removed. Best to my understanding, that was workaround for rendering/updating problems and probably its not needed now. Definitely we shouldn't be adding more auth-related responsibilities to class which are not responsible for authorisation

Reply:
Ah, thanks for the context @wojciech-deriv

Reply:
Have moved those